entity-management-mongo/src/main/java/eu/europeana/entitymanagement/mongo/repository/EntityRecordRepository.java

```diff
@@ -161,6 +162,18 @@ public Optional<EntityRecord> findEntityDupplicationByCoreference(
     EntityRecord value = query.first();
     return Optional.ofNullable(value);
   }
+  
+  public Optional<EntityRecord> findEntityDupplicationBySameAs(String entityId) {
+    List<String> listIds = Collections.singletonList(entityId);
+    Query<EntityRecord> query =
+        getDataStore()
+            .find(EntityRecord.class)
+            .disableValidation()
+            .filter(in(ENTITY_SAME_AS, listIds));
+    query.filter(eq(DISABLED, null));
+    EntityRecord value = query.first();
```

Comment:
we should check if the request returns multiple entities, and probably throw an exception. if we see such cases, we probably have a data issue that needs to be manually corrected in zoho